apps/website/.vuepress/theme/global-components/ReleaseGroup.vue

```diff
@@ -3,11 +3,12 @@
     <h3>{{ title }}</h3>
     <table class="table">
       <tr>
-        <th class="left" style="width: 100px;">Scope</th>
-        <th class="left" style="width: 100px;">Project</th>
+        <th class="left fixed-4">Scope</th>
+        <th class="left fixed-4">Project</th>
```

Comment:
base font size should be "20px" so 4rem is going to look like 80px.

I assume you've got that covered

Reply:
thanks. I am still thinking in 24's from the old ways. 
I'll update and may be able to get rid of the second class.